helsingborg-stad/helsingborg-io-sls-api
# Refactor/ekb collector metrics

Dynamicly creates metrics based on status types.

## How to test

Concrete example:

1. Checkout this branch
2. run `sls deploy` within service folder for `service metrics-api` and `viva-ms`
3. add cases to cases table with attributs GSI2PK set to 'CREATED#202301'
4. set attribute status.type to 'notStarted' or eq (look at possible values in libs/constants.js)
5. hit endpoint /metrics/ekb?format=ebnf
6. verify output
